includes/class-sensei-lesson.php

```diff
@@ -672,36 +690,90 @@ private function save_post_meta( $post_key = '', $post_id = 0 ) {
 			$new_meta_value = ( isset( $_POST[ $post_key ] ) ? sanitize_html_class( $_POST[ $post_key ] ) : '' );
 		} // End If Statement
 
-		// quick edit work around
+		// Quick edit work around.
 		// phpcs:ignore WordPress.Security.NonceVerification
 		if ( 'lesson_preview' == $post_key && isset( $_POST['action'] ) && $_POST['action'] == 'inline-save' ) {
 			$new_meta_value = '-1';
 		}
-		// update field with the new value
+
+		// Get lesson course.
+		if ( '_lesson_prerequisite' === $meta_key ) {
+				$lesson_course_meta = get_post_meta( $post_id, '_lesson_course', true );
+		}
+
+		// phpcs:ignore WordPress.Security.NonceVerification
+		if ( ! empty( $lesson_course_meta ) && isset( $_POST['lesson_course'] ) && $lesson_course_meta !== $_POST['lesson_course'] ) {
```

Comment:
This should be moved inside the previous `if` statement since it's only applicable for lesson prerequisites.

Reply:
Done!